### DISCUSSION TITLE
ScriptReference/Build.Player.ScriptCompilationSettings-extraScriptingDefines

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#663] [Notes]
# ScriptReference/Build.Player.ScriptCompilationSettings-extraScriptingDefines

https://docs.unity3d.com/ScriptReference/Build.Player.ScriptCompilationSettings-extraScriptingDefines.html

**Comment by StinkySteak** [comment]
This extra scripting defines does not override the PlayerSettings, instead it acts as an addition:
Example you have these in player settings:
- DEFINE_ONE
- TWO
- THREE

If you add DEFINE_FOUR to the extraScriptingDefines, then it would be ONE, TWO, THREE, FOUR instead of just FOUR
